README.md

```diff
@@ -165,17 +173,24 @@ Make sure to replace the parameters accordingly.
 10. Trigger the DAG with the following:
 
 ```bash
-airflow trigger_dag -c  "{\"params\": { \"resource_id\": \"my-res-id-123\", \
-\"schema_fields_array\": [ \"field1\", \"field2\" ], \
-\"csv_input\": \"/path/to/my.csv\", \
-\"json_output\": \"/path/to/my.json\" } }" \
-ckan_api_load_single_step
+airflow trigger_dag ckan_api_load_single_step \
+ --conf='{ "resource_id": "my-res-id-123", "schema_fields_array": [ "field1", "field2" ], "csv_input": "/path/to.csv", "json_output": "/path/to.json" }'
 ```
 
 Do not forget to properly replace the parameters with your data and properly escape the special characters.
 Alternatively, you can just run the DAG with the `airflow run` command.
 
-### Example 3: Local file to CKAN DataStore using Postgres
+
+`api_ckan_load_single_node` also works for remote CKAN instances. Just set up your Airflow `CKAN_SITE_URL` variable accordingly.
+
+
+#### Multiple-node DAG
+
+`ckan_api_load_multiple_steps` does the same steps of `api_ckan_load_single_node`, but it uses multiple nodes (tasks). You can repeat the steps of the previous section and run `ckan_api_load_multiple_steps`.
+
```

Comment:
Like in the previous step get the error:
 `ValueError: The execution_date [2020-06-25T06:47:38+00:00] should be >= start_date [2020-06-25T09:47:38.200535+00:00] from DAG's default_args`